app/components/file/report-btn/component.js

```diff
@@ -0,0 +1,278 @@
+import Component from '@glimmer/component';
+import {
+  tracked
+} from '@glimmer/tracking';
+import {
+  task
+} from 'ember-concurrency';
```

Comment:
I think prettier was not run on this file.

Reply:
Yeah, I upgraded the vscode latest version so the config seems reset. I'll fix this in next push